app/views/forms/_odk_questioning.xml.erb

```diff
@@ -0,0 +1,26 @@
+<%# Note that subquestions here refers to multiple levels of a cascading select question, not question grouping. %>
+<% qing.subquestions.each do |subq| %>
+  <%= odk_input_tag(qing, subq, options) do %>
+    <% if options[:appearance] == 'label' %>
+      <label>Choices</label>
+    <% else %>
+      <label ref="jr:itext('<%= subq.odk_code %>:label')"/>
+    <% end %>
+
+    <%# Commcare doesn't like hints. %>
+    <% unless style == 'commcare' or grid_mode %>
```

Comment:
We don't do hints in 'label' case anyway, so this can go altogether for label mode.